packages/documentation-site/patternfly-docs/pages/home.js

```diff
@@ -157,6 +158,9 @@ const FeaturedBlogCard = ({postData, idx}) => {
 
 const HomePage = () => (
   <React.Fragment>
+    <PageSection variant={PageSectionVariants.light} stickyOnBreakpoint={{ default: 'top' }} className="pf-v5-u-font-size-lg">
+      PatternFly 6 alphas are available! Visit the PatternFly 6 staging website (still in in process) for new design, code samples, and documentation. <a href="https://staging-v6.patternfly.org/" className="pf-v5-u-font-weight-bold">Go to PatternFly 6 <ArrowRightIcon /></a>
```

Comment:
```suggestion
      The PatternFly 6 alpha release is available! Our staging website (still under development) has a fresh look, along with new styles, code samples, and documentation. <a href="https://staging-v6.patternfly.org/" className="pf-v5-u-font-weight-bold"> Go to PatternFly 6 <ArrowRightIcon /></a>
```
some rephrasing ideas ~

it could also make sense to remove "(still under development)" and add a separate banner to the staging site? That way the development status context is visible in the location it applies to